Dockerfile.cu118.exllama_multigpu

```diff
@@ -0,0 +1,29 @@
+# NOTE: This Dockerfile is based on CUDA 11.8.
+# To benchmark on other CUDA versions, search and replace "11.8" and "118".
+FROM nvidia/cuda:11.8.0-devel-ubuntu22.04
+
+SHELL ["/bin/bash", "-ec"]
+
+# Step 1. Set up Ubuntu
+RUN grep -v '[ -z "\$PS1" ] && return' ~/.bashrc >/tmp/bashrc                 && \
+    mv /tmp/bashrc ~/.bashrc                                                  && \
+    echo "export EXLLAMA_HOME=/exllama/" >>~/.bashrc                      && \
+    echo "export PATH=/usr/local/cuda/bin/:\$PATH" >>~/.bashrc                && \
+    apt update                                                                && \
+    apt install --yes wget curl git vim build-essential openssh-server cmake
+
+# Step 2. Git clone and compile llama.cpp with cuBLAS
+# NOTE: You may have to tweak `CMAKE_CUDA_ARCHITECTURES`
+# to better accomodate your GPU architecture. It does not usually contribute
+# to performance different though according to our experiments.
+RUN git clone https://github.com/zxybazh/exllama.git $EXLLAMA_HOME       && \
+    cd  exllama                                                          && \
+    git checkout 1d82ed31bb576bf4257add5d6a0a03b396c3b140                && \
+    pip install -r requirements.txt
```

Comment:
You probably don't have to format the benchmark script. Instead just change the lines that are needed for benchmarking. BTW, an alternative to keeping a separate repo is to apply a git-patch, which I suppose would make it more clear that we are using their original implementation